### PR TITLE
Release/v2.3.0

### PR DESCRIPTION
**Pull request by ed-xmos**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
-@Library('xmos_jenkins_shared_library@v0.23.0') _
-
+@Library('xmos_jenkins_shared_library@develop') _
+// New lib checks fn - will be merged into mainline soon so will need to update this tag
```
[the remark this answers is not in the document]
Comment (ed-xmos) [reply Jenkinsfile]: Looks like we can use 0.33.0 so have made that change

[the remark this answers is not in the document]
Comment (ed-xmos) [reply Jenkinsfile]: Actually I lie - it needs JSL @ develop

File: Jenkinsfile [hunk 2]
```diff
@@ -38,21 +36,26 @@ pipeline {
...
+                                        sh 'cmake -B build -G "Unix Makefiles"'
+                                    }
                                 }
                             }
                         }
                     }
                 }
+                stage('Library checks') {
+                    steps {
+                        sh "tree"
```
Comment (shuchitak) [inline Jenkinsfile:55]: did you mean to leave it here?

**Review by shuchitak — DISMISSED** [review]
Looks good!

File: Jenkinsfile (hunk 2 above)
Comment (ed-xmos) [reply Jenkinsfile:55]: Nope - I'll remove it. It's my "where am I" debug method.

**Review by shuchitak — APPROVED** [review]

**Pull request merged**